### DISCUSSION TITLE
Setup Your GitHub Sponsor

### DISCUSSION THREAD

**Discussion by hollowaykeanho** [hollowaykeanho/Upscaler#32] [Q&A] [answered]
@Joly0 , would you mind setup your GitHub Sponsor system up and ready ya? I want to add sponsor button next... to our name =)

**Comment by JeanShuralyov** [comment, marked as answer]
@corygalyna and me are out ya. I know what you're doing =) 

Just place yours and @Joly0 will be suffice.

**Comment by hollowaykeanho** [comment]
noted

**Comment by hollowaykeanho** [comment]
Tracked under issue #40. Closing dicussion.
